hera_sim/tests/test_noise.py

```diff
@@ -33,6 +33,8 @@ def test_resample_Tsky(self):
         # import uvtools, pylab as plt
         # uvtools.plot.waterfall(tsky); plt.show()
 
+### Does not pass, needs to be commented for the pull request
+'''
```

Comment:
`test_noise` failed before I created the `visibility-simulators-GPU` branch (and _still_ fails on the `visibility-simulators` branch).

Reply:
Ah, so it's an old issue. On your local machine, can you navigate to the top-level `hera_sim` directory and run the following sequence of commands?
```
git checkout master
git pull
git checkout visibility-simulators-GPU
git checkout master hera_sim/noise.py hera_sim/tests/test_noise.py
git commit
pytest hera_sim/tests/test_noise.py
```
I believe this should update the `noise` module and the tests that go with it, so that the test shouldn't fail anymore. If it worked, then the tests run with the `pytest` command should pass.

Reply:
OK, so did that, realized I needed to also pull the `interpolators.py`,  `defaults.py` files, the `config` folder, and many of the items in the `data` folder from the master branch (`visibility-simulators` is quite behind it seems...). 

![image](https://user-images.githubusercontent.com/32600543/64298678-46ce9d80-cf2b-11e9-8f0e-975179ecb85c.png)

This is the error I am getting now.

Reply:
Ah, crap, that's right--I made a bunch of additions and modifications to `hera_sim` over the past few months. I'd say just do 
```
git checkout d241bdd hera_sim/noise.py hera_sim/tests/test_noise.py
```
(from the top-level `hera_sim` directory) to revert the changes, then run the tests again and make sure everything passes. Since the `visibility-simulators` branch is so far behind `master`, I think it's OK to leave this test commented out for now. Once we get this branch merged in with `visibility-simulators`, we can merge the differences in `master` and deal with it then.

Reply:
Done.